### PR TITLE
FBR-735 Adjust accrual job to not accrued after early payment.

### PR DESCRIPTION
**Pull request by Deepika1095**
https://fiterio.atlassian.net/browse/FBR-735

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java [hunk 1]
```diff
@@ -1900,15 +1900,16 @@ public Collection<LoanScheduleAccrualData> retrivePeriodicAccrualData(final Loca
...
+                .append(" and (ls.interest_amount <> COALESCE(ls.accrual_interest_derived, 0))))")
                 .append(" and loan.loan_status_id=:active and mpl.accounting_type=:type and (loan.closedon_date <= :tillDate or loan.closedon_date is null)")
                 .append(" and loan.is_npa=false and (ls.duedate <= :tillDate or (ls.duedate > :tillDate and ls.fromdate < :tillDate))) ");
         Map<String, Object> paramMap = new HashMap<>(4);
         if (organisationStartDate != null) {
             sqlBuilder.append(" and ls.duedate > :organisationStartDate ");
             paramMap.put("organisationStartDate", organisationStartDate);
         }
-        sqlBuilder.append(" order by loan.id,ls.duedate ");
+        sqlBuilder.append("  AND loan.id=15868 order by loan.id,ls.duedate ");
```
Comment (leonardohildt) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java]: Please remove the SQL statement for **_AND loan.id =15868_**

**Review by leonardohildt — CHANGES_REQUESTED** [review]
@Deepika1095 please check the comment regarding the loan id in the query

**Review by BrianMuhimbura — APPROVED** [review]

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java (hunk 1 above)
Comment (Deepika1095) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java]: @leonardohildt You are right it was by mistake Thanks

Comment (Deepika1095) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java]: Fixed now

**Review by leonardohildt — APPROVED** [review]

**Pull request merged**